### PR TITLE
Update GCP  quickstart deployment_manager installation docs  for 1.1.0

### PR DESCRIPTION
**Pull request by salrashid123**
Update DM docs for `1.1.0` and add kiali viewer

PR to update the installer:  https://github.com/istio/istio/pull/9695

### CODE REVIEW

File: content/docs/setup/kubernetes/quick-start-gke-dm/index.md [hunk 1]
```diff
@@ -208,7 +209,30 @@ http://localhost:9090/graph
 
 For more details, see [About the Prometheus Add-on](/docs/tasks/telemetry/querying-metrics/#about-the-prometheus-add-on).
 
-### ServiceGraph
+### Kiali and ServiceGraph
+
+Set up a tunnel to Kiali:
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: Please use numbered lists for steps.

File: content/docs/setup/kubernetes/quick-start-gke-dm/index.md [hunk 2]
```diff
@@ -208,7 +209,30 @@ http://localhost:9090/graph
...
+The default username/password for the console is `admin`/`mysecret`.
+
+{{< image width="100%" ratio="53.33%"
+    link="./dm_kiali.png"
+    caption="Kiali"
+    >}}
+
+For more details, see [About the Kiali Add-on](/docs/tasks/telemetry/kiali/).
+
+OR
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: This is not the appropriate way to do it. Please add subheadings for each Kiali and ServiceGraph.
```suggestion
#### ServiceGraph
```

File: content/docs/setup/kubernetes/quick-start-gke-dm/index.md [hunk 3]
```diff
@@ -208,7 +209,30 @@ http://localhost:9090/graph
 
 For more details, see [About the Prometheus Add-on](/docs/tasks/telemetry/querying-metrics/#about-the-prometheus-add-on).
 
-### ServiceGraph
+### Kiali and ServiceGraph
```
Comment (rcaballeromx) [inline content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: You need two headings here. One level three heading that encompasses both Kiali and ServiceGraph. The second heading should then be Kiali. I suggest:
```suggestion
### Install mesh visualization tools

#### Kiali
```

File: content/docs/setup/kubernetes/quick-start-gke-dm/index.md (hunk 1 above)
Comment (salrashid123) [reply content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: for which steps?

Comment (rcaballeromx) [reply content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: All steps.

Comment (salrashid123) [reply content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: done;  that change is pretty simple but this pr isn't meant to address prexisting formatting stuff; if there are more complicated formatting stuff thats needed, please open up a separate an address them there once this is merged

File: content/docs/setup/kubernetes/quick-start-gke-dm/index.md [hunk 4]
```diff
@@ -73,7 +74,7 @@ running and Istio is enabled
     {{< text bash >}}
     $ gcloud container clusters list
     NAME           LOCATION       MASTER_VERSION  MASTER_IP      MACHINE_TYPE   NODE_VERSION  NUM_NODES  STATUS
-    istio-cluster  us-central1-a  1.9.7-gke.1     35.232.222.60  n1-standard-2  1.9.7-gke.1   4          RUNNING
+    istio-cluster  us-central1-a  1.9.7-gke.11    35.188.172.144 n1-standard-1  1.9.7-gke.11  4          RUNNING
```
Comment (sdake) [inline content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: your code has changed to `n1-standard-2`

File: content/docs/setup/kubernetes/quick-start-gke-dm/index.md [hunk 5]
```diff
@@ -208,19 +212,44 @@ http://localhost:9090/graph
...
+{{< image width="100%" ratio="53.33%"
+    link="./dm_kiali.png"
+    caption="Kiali"
+    >}}
+
+For more details, see [About the Kiali Add-on](/docs/tasks/telemetry/kiali/).
+
+#### ServiceGraph
+
+1.  Set up a tunnel to ServiceGraph:
```
Comment (sdake) [inline content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: The TOC recommended a deprecation to servicegraph.  @douglas-reid may have more information.

File: content/docs/setup/kubernetes/quick-start-gke-dm/index.md [hunk 6]
```diff
@@ -166,17 +168,17 @@ If you are using Cloud Shell rather than the installed `gcloud` client, you can
 
 ### Grafana
 
-Set up a tunnel to Grafana:
+1.  Set up a tunnel to Grafana:
```
Comment (sdake) [inline content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: here and below where an ordered list is used please don't add an extra space.  (e.g. this should do `1. Set up a tunnel to Grafana:`

File: content/docs/setup/kubernetes/quick-start-gke-dm/index.md (hunk 5 above)
Comment (salrashid123) [reply content/docs/setup/kubernetes/quick-start-gke-dm/index.md]: set defaults for `ServiceGraph` to false but left code in there to enable it as well as in the docs to state how to access its console

**Comment by geeknoid** [comment]
@rcaballeromx Any more comments on this PR, or can this be merged?

@salrashid123 Just to make sure everything is still sane, would you please rebase and repush. This will run CI tests again.

Thanks.

**Comment by salrashid123** [comment]
@geeknoid 
shoot....i messed up my 2nd rebase (i started with the wrong base as before);  I'll have to cancel this PR out too and move to a cleaner one thats actually uptodate.  I'll send over a new PR and cc you both (and reference this)

**Pull request closed without merging**